lib/runner.js

```diff
@@ -82,7 +82,7 @@ function Runner(opts) {
 	this.sdkVersion = opts.sdk || '3.5.1.GA';
 	this.platform = (opts.platform || '').toLowerCase();
 	/^(?:iphone|ipad)$/.test(this.platform) && (this.platform = 'ios');
-	//this.iosVersion = opts.iosVersion || '7.1';
+	this.iosVersion = opts.iosVersion || '8.1';
```

Comment:
Would `this.iosVersion = opts.iosVersion` be better? 
It's currently used at https://github.com/appcelerator/tio2/blob/master/lib/suite.js#L466 and this would allow it to behave as before when it was commented out and not defaulting to a hardcoded `8.1`.